spec/factories/commercials.rb

```diff
@@ -11,5 +11,7 @@
     factory :event_commercial do
       association :commercialable, factory: :event
     end
+
+    after(:build) { mock_commercial_request }
```

Comment:
Checkout [this comment](https://github.com/openSUSE/osem/pull/786#issuecomment-187099047).